### PR TITLE
Add timezone docs

### LINKED ISSUE #55: Filter for Converting Between Timezones
PR https://github.com/brockpetrie/vue-moment/pull/51 adds `moment-timezone` filters to `vue-moment`.

But I'm not sure how to convert from one timezone to another.

What I want to do is convert from UTC to America/Chicago like so:
```
moment.tz(date, 'UTC').clone().tz('America/Chicago');
```

I would imagine the API would be something like:
```
<span>{{ date | moment('timezone', 'UTC', 'clone', 'timezone', 'America/Chicago') }}</span>
```

Or the API could be smart and allow something like:
```
<span>{{ date | moment('timezone', 'UTC', 'America/Chicago') }}</span>
```

Maybe @BrockReece could answer the question of whether or not this is currently possible and provide a code snippet if it is?

### PR DESCRIPTION
**Pull request by BrockReece**
Add some docs to the readme
Fixes #55

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -134,6 +134,26 @@ Works the same as `add`, but mutates the original moment by subtracting time.
...
+```html
+<span>{{ date | moment('timezone', 'America/Los_Angeles', 'LLLL ss')}}</span>
+```
+
+**To use this filter you will need to pass `moment-timezone` through to the plugin**
+
+```js
+// main.js
+import moment from 'moment-timezone'
+
```
Comment (gbroques) [inline readme.md]: You could add `import VueMoment form 'vue-moment'`.
Might save a newby some confusion.

Comment (BrockReece) [reply readme.md]: Thanks, how does that look?

Comment (gbroques) [reply readme.md]: Looks great. Merge it!

File: readme.md [hunk 2]
```diff
@@ -134,6 +134,26 @@ Works the same as `add`, but mutates the original moment by subtracting time.
 
 For more information about `moment#subtract`, check out <http://momentjs.com/docs/#/manipulating/subtract/>.
 
+### timezone
+
+Convert the date to a certian timezone
```
Comment (gbroques) [inline readme.md]: Typo here with **certian**

Comment (BrockReece) [reply readme.md]: 👍

File: readme.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ For more information about `moment#subtract`, check out <http://momentjs.com/doc
 
 ### timezone
 
-Convert the date to a certian timezone
+Convert the date to a certain timezone
```
Comment (gbroques) [inline readme.md]: Perfect!

**Pull request merged**